bokeh/models/annotations.py

```diff
@@ -183,6 +183,22 @@ class Legend(Annotation):
     when they are drawn.
     """)
 
+    title = String(help="""
```

Comment:
see https://github.com/bokeh/bokeh/blob/master/bokeh/models/annotations.py#L318-L332 for reference